<?xml version="1.0" encoding="UTF-8"?>
<testsuites>
  <testsuite name="fastlane.lanes">
    
    
    
      
      <testcase classname="fastlane.lanes" name="0: cocoapods" time="1.435414">
        
      </testcase>
    
      
      <testcase classname="fastlane.lanes" name="1: git_add" time="0.037013">
        
      </testcase>
    
      
      <testcase classname="fastlane.lanes" name="2: git_commit" time="0.052661">
        
      </testcase>
    
      
      <testcase classname="fastlane.lanes" name="3: push_to_git_remote" time="9.023023">
        
          <failure message="/Library/Ruby/Gems/2.0.0/gems/fastlane-2.25.0/fastlane/lib/fastlane/actions/actions_helper.rb:48:in `execute_action'&#10;/Library/Ruby/Gems/2.0.0/gems/fastlane-2.25.0/fastlane/lib/fastlane/runner.rb:230:in `block in execute_action'&#10;/Library/Ruby/Gems/2.0.0/gems/fastlane-2.25.0/fastlane/lib/fastlane/runner.rb:226:in `chdir'&#10;/Library/Ruby/Gems/2.0.0/gems/fastlane-2.25.0/fastlane/lib/fastlane/runner.rb:226:in `execute_action'&#10;/Library/Ruby/Gems/2.0.0/gems/fastlane-2.25.0/fastlane/lib/fastlane/runner.rb:148:in `trigger_action_by_name'&#10;/Library/Ruby/Gems/2.0.0/gems/fastlane-2.25.0/fastlane/lib/fastlane/fast_file.rb:146:in `method_missing'&#10;Fastfile:21:in `block in parsing_binding'&#10;/Library/Ruby/Gems/2.0.0/gems/fastlane-2.25.0/fastlane/lib/fastlane/lane.rb:33:in `call'&#10;/Library/Ruby/Gems/2.0.0/gems/fastlane-2.25.0/fastlane/lib/fastlane/lane.rb:33:in `call'&#10;/Library/Ruby/Gems/2.0.0/gems/fastlane-2.25.0/fastlane/lib/fastlane/runner.rb:49:in `block in execute'&#10;/Library/Ruby/Gems/2.0.0/gems/fastlane-2.25.0/fastlane/lib/fastlane/runner.rb:45:in `chdir'&#10;/Library/Ruby/Gems/2.0.0/gems/fastlane-2.25.0/fastlane/lib/fastlane/runner.rb:45:in `execute'&#10;/Library/Ruby/Gems/2.0.0/gems/fastlane-2.25.0/fastlane/lib/fastlane/lane_manager.rb:52:in `cruise_lane'&#10;/Library/Ruby/Gems/2.0.0/gems/fastlane-2.25.0/fastlane/lib/fastlane/command_line_handler.rb:30:in `handle'&#10;/Library/Ruby/Gems/2.0.0/gems/fastlane-2.25.0/fastlane/lib/fastlane/commands_generator.rb:104:in `block (2 levels) in run'&#10;/Library/Ruby/Gems/2.0.0/gems/commander-fastlane-4.4.4/lib/commander/command.rb:178:in `call'&#10;/Library/Ruby/Gems/2.0.0/gems/commander-fastlane-4.4.4/lib/commander/command.rb:178:in `call'&#10;/Library/Ruby/Gems/2.0.0/gems/commander-fastlane-4.4.4/lib/commander/command.rb:153:in `run'&#10;/Library/Ruby/Gems/2.0.0/gems/commander-fastlane-4.4.4/lib/commander/runner.rb:476:in `run_active_command'&#10;/Library/Ruby/Gems/2.0.0/gems/fastlane-2.25.0/fastlane_core/lib/fastlane_core/ui/fastlane_runner.rb:38:in `run!'&#10;/Library/Ruby/Gems/2.0.0/gems/commander-fastlane-4.4.4/lib/commander/delegates.rb:15:in `run!'&#10;/Library/Ruby/Gems/2.0.0/gems/fastlane-2.25.0/fastlane/lib/fastlane/commands_generator.rb:303:in `run'&#10;/Library/Ruby/Gems/2.0.0/gems/fastlane-2.25.0/fastlane/lib/fastlane/commands_generator.rb:42:in `start'&#10;/Library/Ruby/Gems/2.0.0/gems/fastlane-2.25.0/fastlane/lib/fastlane/cli_tools_distributor.rb:65:in `take_off'&#10;/Library/Ruby/Gems/2.0.0/gems/fastlane-2.25.0/bin/fastlane:20:in `&lt;top (required)&gt;'&#10;/usr/bin/fastlane:22:in `load'&#10;/usr/bin/fastlane:22:in `&lt;main&gt;'&#10;&#10;Exit status of command 'git push origin master:master --tags' was 1 instead of 0.&#10;To https://github.com/winsions/XMGRemotePlayer.git&#10; * [new tag]         0.1.0 -&gt; 0.1.0&#10; * [new tag]         0.1.1 -&gt; 0.1.1&#10; ! [rejected]        master -&gt; master (fetch first)&#10;error: failed to push some refs to 'https://github.com/winsions/XMGRemotePlayer.git'&#10;hint: Updates were rejected because the remote contains work that you do&#10;hint: not have locally. This is usually caused by another repository pushing&#10;hint: to the same ref. You may want to first integrate the remote changes&#10;hint: (e.g., 'git pull ...') before pushing again.&#10;hint: See the 'Note about fast-forwards' in 'git push --help' for details.&#10;" />
        
      </testcase>
    
  </testsuite>
</testsuites>
